From wfp5p@virginia.edu  Tue Jul 28 14:50:37 2009
From: Bill Pemberton <wfp5p@virginia.edu>
Date: Tue, 28 Jul 2009 13:46:23 -0400
Subject: Staging: hv: comment out blkdev variable in blkvsc_ioctl
To: greg@kroah.com
Cc: hjanssen@microsoft.com
Message-ID: <1248803186-1909-3-git-send-email-wfp5p@virginia.edu>


This variable generated an unused variable warning due to other code
in the fuction being commented out.  This comments out the variable
defination so that the code compiles without warnings.

Signed-off-by: Bill Pemberton <wfp5p@virginia.edu>
Cc: Hank Janssen <hjanssen@microsoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/hv/blkvsc_drv.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/hv/blkvsc_drv.c
+++ b/drivers/staging/hv/blkvsc_drv.c
@@ -1461,7 +1461,7 @@ int blkvsc_getgeo(struct block_device *b
 static int blkvsc_ioctl(struct block_device *bd, fmode_t mode,
 			unsigned cmd, unsigned long argument)
 {
-	struct block_device_context *blkdev = bd->bd_disk->private_data;
+/*	struct block_device_context *blkdev = bd->bd_disk->private_data; */
 	int ret=0;
 
 	switch (cmd)
